src/core/cardano/walletConnect/identityWalletConnect.ts

```diff
@@ -30,16 +31,17 @@ class IdentityWalletConnect extends CardanoPeerConnect {
       discoverySeed: discoverySeed,
       logLevel: "info",
     });
-
+    this.selectedAid = selectedAid;
     this.signerCache = new Map();
 
-    this.getIdentifierOobi = async (): Promise<string> => {
-      const identifiers = await Agent.agent.identifiers.getIdentifiers();
-      if (!(identifiers && identifiers.length > 0)) {
-        throw new Error(IdentityWalletConnect.NO_IDENTIFIERS_STORED);
+    this.getIdentifierOobi = async (identifierAid: string): Promise<string> => {
```

Comment:
This function will be called by the dApp - it won't know the `identifierAid` yet.

You are supposed to call `await Agent.agent.identifiers.getIdentifier(this.selectedAid);`